cs/libdpr/samples/DprCounters/DprCounters/scripts/kubectl_stop.sh

```diff
@@ -0,0 +1,10 @@
+kubectl delete svc counter-server-svc-0
```

Comment:
Just to be clear: this is specific for a deployment and not a generic script right?

Reply:
Yes, this script exists from way back when and I still use it because it is convenient. It is not actual "production" material. Eventually we will use the python code to safely dismantle the DPR Cluster.